shared/common-adapters/labeled-input.tsx

```diff
@@ -94,6 +94,7 @@ const ReflessLabeledInput = (props: Props & RefProps) => {
         ref={props.forwardedRef}
         style={Styles.collapseStyles([
           styles.input,
+          props.style,
```

Comment:
This was not being plumbed through erroneously, I added it and removed the instances of this prop we had before